bin/phpmnd

```diff
@@ -36,4 +36,28 @@ if (false === $loaded) {
 ini_set('xdebug.max_nesting_level', 10000);
```

Comment:
Yeah, it doesn't sit right here, but can't think of a better way via Symfony/Console. We're basically checking if no command is sent, and prefixing `run`. We have a `run` command since adding multiple commands (even two for self-update) seems to require abandoning a default command. Also, the using `setDefaultCommand` method for Application doesn't accept arguments so that's not a solution.

Not sure if these are specific design decisions (or just limitations) by the Symfony team, or I'm just missing something obvious that would make that `prepareArgv` function go away.